Product/Production/Adapters/AdminDistribution_a0/pom.xml

```diff
@@ -20,6 +20,18 @@
             <artifactId>AdminDistributionCore</artifactId>
             <version>${project.parent.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.connectopensource</groupId>
+            <artifactId>CONNECTCoreLib</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.connectopensource</groupId>
+            <artifactId>CONNECTCommonTypesLib</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.connectopensource</groupId>
+            <artifactId>AdminDistributionWebservices</artifactId>
+        </dependency>
     </dependencies>
```

Comment:
Originally, every module under "Adapters" inherited CONNECTCoreLib and CONNECTCommonTypesLib (amongst other dependencies) from the Adapters POM.  This is a bad practice (POM modules don't have dependencies), and was causing problems during CXF unbundling due to transitive dependencies.

Instead, each POM now declares, to the best of my ability, exactly what it needs -- no more, no less.